### DISCUSSION TITLE
[Connector Request] MetaMask connector to support MetaMask SDK

### DISCUSSION THREAD

**Discussion by andreahaku** [wagmi-dev/references#293] [Connector request]
### What **novel use-case** does the Connector provide?

We are MetaMask SDK developers and we'd like to add support for the MetaMask SDK to the MetaMask connector.

This would enable to add support to MetaMask Mobile app along with MetaMask Extension.

The MetaMask Mobile connection would be enabled on both desktop browsers (via QRCode scanning) and mobile browsers (via Universal Links).

### Are the Connector's integrations production-ready and generally available?

the update will add the '@metamask/sdk' dependency which is the MetaMask SDK npm module.

### Are you committed to actively mainting the Connector?

- [X] Yes, my team is or I am committed to actively maintaining the Connector.

**Comment by tmm** [comment]
Sounds great! Go for it.
